scripts/zowe-install-zlux.sh

```diff
@@ -55,9 +63,11 @@ cp -f ${INSTALL_DIR}/files/zlux/config/plugins/* zlux-app-server/defaults/plugin
 
 echo "Unpax zssServer " >> $LOG_FILE
 cd zlux-app-server
-pax -r -px -f $INSTALL_DIR/files/zss.pax bin
+pax -r -px -f $INSTALL_DIR/files/zss.pax ${ZSS_COMPONENT_DIR}
```

Comment:
need to test to confirm if this places samplib/loadlib in a duplicate spot or if that is the intended location